recipes/lapack/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+        return cmake
+
+    def build(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("This library cannot be built with Visual Studio. Please use MinGW to "
+                            "build it and option 'visual_studio=True' to build and consume.")
+        cmake = self._configure_cmake()
+        for target in ["blas", "cblas", "lapack", "lapacke"]:
+            cmake.build(target=target)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+        if self.options.visual_studio:
+            for bin_path in self.deps_cpp_info.bin_paths:  # Copy MinGW dlls for Visual Studio consumers
+                self.copy(pattern="*seh*.dll", dst="bin", src=bin_path, keep_path=False)
+                self.copy(pattern="*sjlj*.dll", dst="bin", src=bin_path, keep_path=False)
+                self.copy(pattern="*dwarf2*.dll", dst="bin", src=bin_path, keep_path=False)
+                self.copy(pattern="*quadmath*.dll", dst="bin", src=bin_path, keep_path=False)
+                self.copy(pattern="*winpthread*.dll", dst="bin", src=bin_path, keep_path=False)
+                self.copy(pattern="*gfortran*.dll", dst="bin", src=bin_path, keep_path=False)
+
+            with tools.chdir(os.path.join(self.package_folder, "lib")):
+                libs = glob.glob("lib*.a")
+                for lib in libs:
+                    vslib = lib[3:-2] + ".lib"
+                    self.output.info('renaming %s into %s' % (lib, vslib))
+                    shutil.move(lib, vslib)
+
+    def package_id(self):
+        if self.options.visual_studio:
+            self.info.settings.compiler = "Visual Studio"
+            del self.info.settings.compiler.version
+            del self.info.settings.compiler.runtime
+            del self.info.settings.compiler.toolset
+
+    def package_info(self):
+        # the order is important for static builds
+        self.cpp_info.libs = ["lapacke", "lapack", "blas", "cblas"]
+        self.cpp_info.system_libs.extend(["gfortran", "m"])
+        if self.options.visual_studio and self.options.shared:
+            self.cpp_info.libs = ["lapacke.dll.lib", "lapack.dll.lib", "blas.dll.lib", "cblas.dll.lib"]
+        self.cpp_info.libdirs = ["lib"]
+        if tools.os_info.is_macos:
+            brewout = StringIO()
+            try:
+                self.run("gfortran --print-file-name libgfortran.dylib", output=brewout)
+            except Exception as error:
+                raise ConanException("Failed to run command: {}. Output: {}".format(error, brewout.getvalue()))
+            lib = os.path.dirname(os.path.normpath(brewout.getvalue().strip()))
+            self.cpp_info.libdirs.append(lib)
```

Comment:
When a fortran compiler gets added to cci, it should add its library directories in its own `package_info` method.

```suggestion
```